### ISSUE TITLE
Naming convention for modified standard recipes

### ISSUE THREAD

**Issue by TomasTorsvik** [NorESMhub/noresmvaltool#1]
How to name modified standard recipes in the most sensible way?

The "tested_recipes" directory contains some recipes that have been modified based on standard recipes. I think it makes sense to keep the main part of the recipe name and add a label, i.e. <standard_recipe_name>_<new_label>.yml, but it's not clear what is the best approach. Here is a suggestion based on some typical situations

1. The recipe compares a single model output with observational data. In this case we could use the model name as label
   ex.: recipe_modes_of_variability.yml  -->  recipe_modes_of_variability_NorESM1-M.yml
2. A recipe constructs a multi-model product. In this case we probably want dataset from the same CMIP
   ex.: recipe_ocean_example.yml  -->  recipe_ocean_example_CMIP5.yml
  - Problem: "CMIP5" is already used as a label in perfmetrics recipes, maybe "CMIP6" will also appear, causing confusion between standard and modified recipes 
3. A recipe may contain dataset from both CMIP5 and CMIP6, in particular if we want to look at the evolution of model performance, so CMIP5/6  may not be appropriate in this case.
 
NorESM1 and NorESM2 does not correspond to CMIP5 and CMIP6, respectively (CMIP6 contains NorESM1-F datasets), so this can be confusing as well.

Are there other labels we could use that would make more sense?

**Comment by YanchunHe** [comment]
The ESMValTool is designed for multiple models comparison (>=1), and can work across CMIP5/CMIP6 dataset. Therefore, it is not sensible to have model name or CMIP5/6 as suffix in the recipe names. Except in the case that it is not able to combine different models (which should not be the case) or CMIP phases (that is possible at some point).

Therefore, I would not have any suffix. It is good to include as many as possible different model output and observational data activated in the tested recipes, but it can also also included (activated) one or several models. There are many options in one recipe, so it is the end-users' choice to switch on and off some of them. We put the 'tested_recipes" so that it can run without any modification.  They are just a start for other users to adapt to their own needs, e.g., use more model, switch on/off additional diagnostics.

In some cases, we can have, e.g., CMIP5/CMIP6 in the suffix, as I found that it may take quite some time to combine them, as the ESMValTool does not support this well. In the long run, this should also be fixed.

In short, I would suggest no suffix is used but keep the same name as the standard recipes; Use only CMIP5/CMIP6 is it is necessary.
